CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -446,17 +446,10 @@ CF_EXPORT double kCFCoreFoundationVersionNumber;
 #define kCFCoreFoundationVersionNumber_iOS_9_x_Max 1299
 #endif
 
-#if __LLP64__
-typedef unsigned long long CFTypeID;
-typedef unsigned long long CFOptionFlags;
-typedef unsigned long long CFHashCode;
-typedef signed long long CFIndex;
-#else
-typedef unsigned long CFTypeID;
-typedef unsigned long CFOptionFlags;
-typedef unsigned long CFHashCode;
-typedef signed long CFIndex;
-#endif
+typedef uintptr_t CFTypeID;
+typedef uintptr_t CFOptionFlags;
```

Comment:
This affects Darwin.

@parkera @phausler Do you see any scenario in which this wouldn’t be strictly equivalent?

Reply:
are we in any danger with C++? or swift imports here?

This is a public API change so lets be super careful here.

Reply:
I suppose that we could just map these specifically in the clang-importer.  I think that we should talk to @jrose-apple about that.

I suppose, that this *could* cause issue where there are template specializations using `CFTypeID`, `CFOptionsFlags`.